integration-web/src/main/resources/application.yml

```diff
@@ -59,4 +59,11 @@ management:
 #
 # Time (in minutes) to persist in a local cache the pod certificate to check JWT signature
 #
-public_pod_certificate_cache_duration: 60
\ No newline at end of file
+public_pod_certificate_cache_duration: 60
+
+#
+# Health check timeouts
+#
+health_check_timeouts:
+  connect_timeout_in_millis: 1000
```

Comment:
I think we should increase this timeout to 2000 like other API calls.

Reply:
Done.